pages/platform/kubernetes-k8s/known-limits/guide.en-asia.md

```diff
@@ -47,6 +47,12 @@ Most worker nodes (be them added manually or through cluster autoscaler) are cre
 
 Delivering a fully managed service, including OS and other component updates, you will neither need nor be able to SSH as root into your nodes.
 
+## Data persistence
+
+If an incident is detected by the OVHcloud monitoring, as part of autohealthing, or in case of a version upgrade, the Nodes can be fully reinstalled. 
```

Comment:
Hi @scraly, I sugget "auto-healing" instead of autohealthing. Please update on all versions.

Reply:
Thanks, fixed for all locales/versions :)